src/compiler/clang.rs

```diff
@@ -113,6 +113,8 @@ counted_array!(pub static ARGS: [ArgInfo<gcc::ArgData>; _] = [
     take_arg!("-include-pch", PathBuf, CanBeSeparated, PreprocessorArgumentPath),
     take_arg!("-load", PathBuf, Separated, ExtraHashFile),
     take_arg!("-mllvm", OsString, Separated, PassThrough),
+    take_arg!("-plugin-arg-blink-gc-plugin", OsString, Separated, PassThrough),
+    take_arg!("-plugin-arg-find-bad-constructs", OsString, Separated, PassThrough),
```

Comment:
If we simply use `Concatenated` to capture the plugin name, additional logic is needed to consume the argument value.
```suggestion
    take_arg!("-plugin-arg", OsString, Concatenated('-'), PassThrough),
```

```
running 2 tests
test compiler::clang::test::test_parse_xclang_plugin_arg_blink_gc_plugin ... FAILED
test compiler::clang::test::test_parse_xclang_plugin_arg_find_bad_constructs ... FAILED

failures:

---- compiler::clang::test::test_parse_xclang_plugin_arg_blink_gc_plugin stdout ----
thread 'compiler::clang::test::test_parse_xclang_plugin_arg_blink_gc_plugin' panicked at 'Got unexpected parse result: CannotCache("Can\'t handle Raw arguments with -Xclang", None)', src/compiler/clang.rs:285:17

---- compiler::clang::test::test_parse_xclang_plugin_arg_find_bad_constructs stdout ----
thread 'compiler::clang::test::test_parse_xclang_plugin_arg_find_bad_constructs' panicked at 'Got unexpected parse result: CannotCache("Can\'t handle Raw arguments with -Xclang", None)', src/compiler/clang.rs:316:17
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```